portfolio/src/main/webapp/style.css

```diff
@@ -105,3 +106,62 @@ div#content {
   }
 }
 /*End styles for vertical navigation bar*/
+
+
+/* Begin box design for work and projects */
+* {
+    box-sizing: border-box;
+}
+
+.main {
+  max-width: 1000px;
+  margin: auto;
+}
+
+.row {
+  margin: 8px -16px;
+  display: inline-flex;
+background-color: #f1f1f1;
```

Comment:
Nit: Fix the indentation.

Reply:
Fixed by ensuring CSS indentation is all aligned, and consistently used tabs